mindsdb/api/mongo/responders/aggregate.py

```diff
@@ -61,9 +61,20 @@ def result(self, query, request_env, mindsdb_env, session):
         db = query['$db']
         collection = query['aggregate']
 
-        ast_query = aggregate_to_ast(query, request_env.get('database', 'mindsdb'))
+        first_step = query['pipeline'][0]
+        if '$match' in first_step:
+            ast_query = aggregate_to_ast(query, request_env.get('database', 'mindsdb'))
 
-        data = run_sql_command(request_env, ast_query)
+            data = run_sql_command(request_env, ast_query)
+
+        elif '$collStats' in first_step:
+            raise NotImplementedError(
+                "To describe model use:"
+                " db.runCommand( { collStats: 'model_name', scale: 'describe_type'})"
```

Comment:
maybe also add a link to the doc? https://docs.mindsdb.com/sdks/mongo/models/describe

Reply:
docs updated